### PR TITLE
Issue #3897: Splitted input test files for WhitespaceAfterCheckTest

### PR DESCRIPTION
**Pull request by Pedro12909**
https://github.com/checkstyle/checkstyle/issues/3897

### CODE REVIEW

**Comment by rnveach** [comment]
@Pedro12909 CIs need to be green.
TC Error is:
````
WhitespaceAfterCheckTest.java (1)
40: filePrefix Field filePrefix may be 'final'
````

I think the field `filePrefix` is excessive for test area. It makes the tests harder to read and track down the input file.
Please remove it and just leave the full file name. Ex: `getPath("InputWhitespaceAfterSimple.java")`

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java [hunk 1]
```diff
@@ -106,7 +111,8 @@ public void testLiteralWhile() throws Exception {
         final String[] expected = {
             "39:14: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "while"),
         };
-        verify(configurationTestLiteralWhile, getPath("InputWhitespaceAfter.java"), expected);
+        verify(configurationTestLiteralWhile, getPath("InputWhitespaceAfterConditions.java"),
```
Comment (MEZk) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Please make separate input file for each test.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java [hunk 2]
```diff
@@ -117,7 +123,8 @@ public void testLiteralIf() throws Exception {
         final String[] expected = {
             "18:11: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "if"),
         };
-        verify(configurationTestLiteralIf, getPath("InputWhitespaceAfter.java"), expected);
+        verify(configurationTestLiteralIf, getPath("InputWhitespaceAfterConditions.java"),
```
Comment (MEZk) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Please make separate input file for each test.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java [hunk 3]
```diff
@@ -128,7 +135,8 @@ public void testLiteralElse() throws Exception {
         final String[] expected = {
             "27:15: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "else"),
         };
-        verify(configurationTestLiteralElse, getPath("InputWhitespaceAfter.java"), expected);
+        verify(configurationTestLiteralElse, getPath("InputWhitespaceAfterConditions.java"),
```
Comment (MEZk) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Please make separate input file for each test.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java [hunk 4]
```diff
@@ -139,7 +147,8 @@ public void testLiteralFor() throws Exception {
         final String[] expected = {
             "51:12: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "for"),
         };
-        verify(configurationTestLiteralFor, getPath("InputWhitespaceAfter.java"), expected);
+        verify(configurationTestLiteralFor, getPath("InputWhitespaceAfterConditions.java"),
```
Comment (MEZk) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Please make separate input file for each test.

Comment (Pedro12909) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Shouldn't testLiteralIf() and testLiteralElse() use the same input file?

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java [hunk 5]
```diff
@@ -104,9 +109,10 @@ public void testLiteralWhile() throws Exception {
                 createCheckConfig(WhitespaceAfterCheck.class);
         configurationTestLiteralWhile.addAttribute("tokens", "LITERAL_WHILE");
         final String[] expected = {
-            "39:14: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "while"),
+            "16:14: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "while"),
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: We are not expecting line numbers to change. Files should be like 99% direct copy/move.
Package name and class name can be changed if it is not dependent on test. All others should stay as is.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java [hunk 6]
```diff
@@ -115,9 +121,10 @@ public void testLiteralIf() throws Exception {
                 createCheckConfig(WhitespaceAfterCheck.class);
         configurationTestLiteralIf.addAttribute("tokens", "LITERAL_IF");
         final String[] expected = {
-            "18:11: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "if"),
+            "16:11: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "if"),
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Same.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java [hunk 7]
```diff
@@ -126,9 +133,10 @@ public void testLiteralElse() throws Exception {
                 createCheckConfig(WhitespaceAfterCheck.class);
         configurationTestLiteralElse.addAttribute("tokens", "LITERAL_ELSE");
         final String[] expected = {
-            "27:15: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "else"),
+            "20:15: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "else"),
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Same.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java [hunk 8]
```diff
@@ -137,9 +145,10 @@ public void testLiteralFor() throws Exception {
                 createCheckConfig(WhitespaceAfterCheck.class);
         configurationTestLiteralFor.addAttribute("tokens", "LITERAL_FOR");
         final String[] expected = {
-            "51:12: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "for"),
+            "16:12: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "for"),
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Same.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java [hunk 9]
```diff
@@ -148,9 +157,10 @@ public void testLiteralDo() throws Exception {
                 createCheckConfig(WhitespaceAfterCheck.class);
         configurationTestLiteralDo.addAttribute("tokens", "LITERAL_DO");
         final String[] expected = {
-            "63:11: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "do"),
+            "16:11: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "do"),
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Same.

**Comment by rnveach** [comment]
@Pedro12909 Please also squash all changes to 1 commit. We will not merge with multiple.

**Comment by rnveach** [comment]
@Pedro12909 Please respond to done to each review item as we don't get notifications when you push new changes. It also helps us reverify each item was done.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java (hunk 8 above)
Comment (Pedro12909) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Done.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java (hunk 9 above)
Comment (Pedro12909) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Done.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java (hunk 7 above)
Comment (Pedro12909) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Done.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java (hunk 6 above)
Comment (Pedro12909) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Done.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java (hunk 5 above)
Comment (Pedro12909) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Done.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java (hunk 1 above)
Comment (Pedro12909) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Done.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java (hunk 2 above)
Comment (Pedro12909) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Done.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java (hunk 3 above)
Comment (Pedro12909) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Done.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java (hunk 4 above)
Comment (Pedro12909) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Done.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java [hunk 10]
```diff
@@ -71,7 +73,8 @@ public void testCast() throws Exception {
         final String[] expected = {
             "88:21: " + getCheckMessage(MSG_WS_TYPECAST),
         };
-        verify(configurationTestCast, getPath("InputWhitespace.java"), expected);
+        verify(configurationTestCast, getPath("InputWhitespaceAfter.java"),
```
Comment (MEZk) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: InputWhitespaceAfterTypecast

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java [hunk 11]
```diff
@@ -60,7 +61,8 @@ public void testDefault() throws Exception {
             "42:40: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, ","),
             "71:30: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, ","),
         };
-        verify(checkConfig, getPath("InputSimple.java"), expected);
+        verify(checkConfig, getPath("InputWhitespaceAfterSimple.java"),
```
Comment (MEZk) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: InputWhitespaceAfterDefaultConfig

Comment (Pedro12909) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Done

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java (hunk 10 above)
Comment (Pedro12909) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java]: Done

**Pull request merged**